### ISSUE TITLE
Repository Info in Fossil Sample Resource Model and Details Card

### ISSUE THREAD

**Issue by LeannePyle** [bcgov/BCHeritage#756]
- [x] Move Repository Info up in the list to display first

**Comment by bferguso** [comment]
Also moved the Fossil ID and Fossil Occurance Site ID to the bottom. (Should these be on the same card as they ar only for historical values?)

**Comment by LeannePyle** [comment]
I do not think we will ever use or reference these - only for historic values - where will they go?

**Comment by bferguso** [comment]
This is the repository where the sample is stored at, including repository number and field number, etc. This is used as the primary descriptor and is how a fossil sample is identified.

**Comment by bferguso** [comment]
Automated cleanup
